build.sh

```diff
@@ -1,12 +1,24 @@
-#!/bin/bash
+#!/usr/bin/env bash
 
-BASEDIR=$(pwd)
+set -e
+
+follow_link() {
+  FILE="$1"
+  while [ -h "$FILE" ]; do
+    # On Mac OS, readlink -f doesn't work.
+    FILE="$(readlink "$FILE")"
+  done
+  echo "$FILE"
+}
+
+SCRIPT_PATH=$(realpath "$(dirname "$(follow_link "$0")")")
```

Comment:
May require a mention in the prerequisites. 

For example, on MacOS: `brew install coreutils` would be typically be required.

Ryan

Reply:
Added to the README.md.